tests/asynctests/combined_asynctests.js

```diff
@@ -0,0 +1,44 @@
+
+function testAll() 
+{
+
+    api.getIntValueAsync( function (val) {
+            compare(val,10);
+            });
+
+    api.getStringValueAsync( function (val) {
+            compare(val,"Test-String" );
+            });
+
+    
+    api.getContainerValueAsync( function (val) {
+            compare(val.idata,9);
+            compare(val.sdata,"Test-String");
+            // compare(val.mdata["Test-String"],27);
+            });
+
+
+
+    // asynchronous call
+    api.calculateSumAsync(1,2,3,function (val){
+        compare(6,val);
+    });
+
+
+    api.append("Rakesh","Sadhu",function (val){
+        compare(val,"RakeshSadhu");
+    });
+
+
+
+
+
+    //synchronous call 
```

Comment:
What's the reason for synchronous function tests? Doesn't really make sense in async test case.